core/src/main/scala/chisel3/internal/Builder.scala

```diff
@@ -485,7 +485,7 @@ private[chisel3] object Builder extends LazyLogging {
   }
 
   // Returns the prefix stack at this moment
-  def getPrefix(): Prefix = chiselContext.get().prefixStack
+  def getPrefix: Prefix = chiselContext.get().prefixStack
```

Comment:
why does this one not need a deprecation version?

Reply:
Because it's [package] private, we can do whatever we want. That being said, this includes a `.get()` of the dynamic context so I'd say keep the `()`. 

```suggestion
  def getPrefix(): Prefix = chiselContext.get().prefixStack
```